.circleci/config.yml

```diff
@@ -198,7 +199,7 @@ workflows:
   security:
     triggers:
       - schedule:
-          cron: "5 3 * * 1-5"
+          cron: "30 5 * * 1-5"
```

Comment:
I think these cron timings should remain as they were? I believe all projects have different times,

Reply:
I agree - resolved in the subsequent commit